packages/react/src/popover/popup/PopoverPopup.tsx

```diff
@@ -126,7 +125,10 @@ export const PopoverPopup = React.forwardRef(function PopoverPopup(
 
   // If there's just one trigger, we can skip the auto-resize logic as
   // the popover will always be anchored to the same position.
-  const autoresizeEnabled = () => store.context.triggerElements.size > 1;
+  const autoresizeEnabled = React.useCallback(
+    () => store.context.triggerElements.size > 1,
+    [store],
+  );
```

Comment:
Maybe it should check if animations are enabled on the positioner? Or if it has fixed size variables? e.g. if detached triggers are being used without transitions/fixed sizes, does this still work?